From 21da35d694d368a713b5e228211053a749ae253e Mon Sep 17 00:00:00 2001
From: Julien Vuillaumier <julien.vuillaumier@nxp.com>
Date: Thu, 14 Oct 2021 10:20:38 +0200
Subject: [PATCH] [armnn] compilation error - unused parameter

correction for occurences of compilation error below, when compilation
switch ENABLE_ARMNN_CAFFE is disabled.

error: unused parameter '[parameter name]' [-Werror=unused-parameter]

Upstream-Status: Accepted for 2.1.0
(https://github.com/nnstreamer/nnstreamer/commit/d7265ed56a9133324a1411293cc69939bba1f364)

Signed-off-by: Julien Vuillaumier <julien.vuillaumier@nxp.com>
---
 ext/nnstreamer/tensor_filter/tensor_filter_armnn.cc | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/ext/nnstreamer/tensor_filter/tensor_filter_armnn.cc b/ext/nnstreamer/tensor_filter/tensor_filter_armnn.cc
index 0fcc72f9..b1762a52 100644
--- a/ext/nnstreamer/tensor_filter/tensor_filter_armnn.cc
+++ b/ext/nnstreamer/tensor_filter/tensor_filter_armnn.cc
@@ -227,6 +227,9 @@ int
 ArmNNCore::makeCaffeNetwork (std::map<std::string, armnn::TensorShape> &input_map,
     std::vector<std::string> &output_vec)
 {
+  UNUSED (input_map);
+  UNUSED (output_vec);
+
   g_printerr ("ARMNN-CAFFE was not enabled at build-time. tensor-filter::armnn cannot handle caffe networks.");
   return -EPERM;
 }
-- 
2.25.1

